src/components/apps/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'preact/hooks';
+import clsx from 'clsx';
+import { AppIcon } from '__/components/utils/AppIcon';
+import css from './Calendar.module.scss';
+import { YearView } from './Views/YearView';
+import { WeekView } from './Views/WeekView';
+import { MonthView } from './Views/MonthView';
+import { DayView } from './Views/DayView';
+import dayjs from 'dayjs';
+import { CalendarAppContext } from './context';
+import { mdiChevronLeft, mdiChevronRight } from '@mdi/js';
+
+enum VIEW_OPTIONS {
+  YEAR,
+  MONTH,
+  WEEK,
+  DAY,
+}
+
+export const Calendar = () => {
+  const [view, setView] = useState(VIEW_OPTIONS.MONTH);
+  const [selectedDate, setSelectedDate] = useState(dayjs());
```

Comment:
Would you like to consider to use `dayjs` instead? I saw from the current code, there is only one place that `date-fns` is used. If you are ok, I can help to migrate the current one into `dayjs`.

Reply:
Hmmm dayjs is great but I'd prefer if we could stick to date-fns. I prefer to have a buy-in(imports in date-fns) rather than have it all be in at once(dayjs as constructor).

Reply:
That's alright. Done migration